rasa/__main__.py

```diff
@@ -122,6 +122,7 @@ def main() -> None:
             rasa.telemetry.initialize_telemetry()
             rasa.telemetry.initialize_error_reporting()
             plugin_manager().hook.init_telemetry(endpoints_file=endpoints_file)
+            plugin_manager().hook.initialize_managers(endpoints_file=endpoints_file)
```

Comment:
just noticed the inconsistency with `init_telemetry` 💀  - but that ship sailed 😂

Reply:
wait, no the others aren't hookspecs. should we then reuse the `init_` prefix here?

Reply:
Yup I can make that change, we just need to change the `hookimpl` in `rasa_plus` too. cc @radovanZRasa